qualtran/bloqs/reflections/reflection_using_prepare.py

```diff
@@ -126,3 +153,30 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> cirq.Circ
         wire_symbols = ['@' if self.control_val else '@(0)'] * total_bits(self.control_registers)
         wire_symbols += ['R_L'] * total_bits(self.selection_registers)
         return cirq.CircuitDiagramInfo(wire_symbols=wire_symbols)
+
+
+@bloq_example(generalizer=ignore_split_join)
```

Comment:
Can you also add a `build_call_graph` so we support symbolics?

Reply:
I'm also thinking we should update the wire_symbols to potentially special case to the nice open / closed circle diagram if the reflection is a reflect around zero, but that's not super high priority.